docs/terms/expectation.md

```diff
@@ -3,10 +3,134 @@ title: Expectation
+2. Expertise. Rich experience from subject matter experts, Analysts, and data owners is often a critical source of Expectations. Interviewing experts and encoding their tacit knowledge of common distributions, values, or failure conditions can be can excellent way to generate Expectations.
+3. Exploratory Analysis. Using Great Expectations in an exploratory analysis workflow (e.g. within Jupyter Notebooks)is an important way to develop experience with both raw and derived datasets and generate useful and testable Expectations about characteristics that may be important for the data's eventual purpose, whether reporting or feeding another downstream model or data system.
+
+For more information on these methods, please see our [overview guide for creating Expectations](../guides/expectations/create_expectations_overview.md#the-create-expectations-process).
+
+### Configuration
+
+Most of the time you will not need to directly interact with an Expectation's configurations.  However, advanced users may have circumstances in which it is desirable to define Expectations based purely on domain knowledge, without comparing against underlying data.  To do this, you will need to directly write an Expectation's configuration.  For details on how to do this, please reference our guide on [how to create and edit Expectations based on domain knowledge, without inspecting data directly](../guides/expectations/how_to_create_and_edit_expectations_based_on_domain_knowledge_without_inspecting_data_directly.md).
+
+The other occasion when you may want to edit an Expectation's configuration is when you need to edit the `result_format` of a <TechnicalTag relative="../" tag="custom_expectation" text="Custom Expectation" />  The `result_format` parameter may be either a string or a dictionary which specifies the fields to return in `result`.  For further details, please see [our reference guide on the result_format parameter](../reference/expectations/result_format.md).
+
+### Results
+
+All Expectations return a JSON-serializable dictionary when evaluated, which consists of four standard (though optional, depending on the type of Expectation in question) arguments.  These are: `result_format`, `include_config`, `catch_exceptions`, and `meta`.  For a more detailed explanation as to what each of these arguments consists of and which Expectations use them, please see [our reference guide on standard arguments](../reference/expectations/standard_arguments.md).
+
+## More details
+
+### Expectation Concepts: Domain and Success Keys
+
+A **domain** makes it possible to address a specific set of data, such as a *table*, *query result*, *column* in a table or dataframe, or even a Metric computed on a previous Batch of data.
```

Comment:
This definition seems a bit confusing, but I'm having a hard time thinking of something better.

The way I think of it, a domain describes the locale of data to which a metric or expectation applies. I don't love this definition either, though...

Reply:
I've added your definition as an expansion to the existing one.